<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogOpenDev</class>
 <widget class="QDialog" name="DialogOpenDev">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>打开RTL设备</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>480</width>
     <height>218</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labHint">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>未找到RTL设备，请插入RTL设备后再尝试刷新按钮！</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetDevice"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QToolButton" name="btnRefresh">
        <property name="toolTip">
         <string>刷新可用设备</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="mainwindow.qrc">
          <normaloff>:/icon/Resources/refresh.ico</normaloff>:/icon/Resources/refresh.ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOK">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>DialogOpenDev</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DialogOpenDev</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
